app/authorization_test.go

```diff
@@ -30,7 +31,7 @@ func TestCheckIfRolesGrantPermission(t *testing.T) {
 
 	for testnum, testcase := range cases {
 		if th.App.RolesGrantPermission(testcase.roles, testcase.permissionId) != testcase.shouldGrant {
```

Comment:
Here I would use `assert` instead of `require` because that way can be executed all the cases and if not, it will stop in the first wrong case. Besides, part of the effort is to convert the assertion in a more "semantic" assertions, in this case something like:

```go
assert.Equal(t, th.App.RolesGrantPermission(testcase.roles, testcase.permissionId), testcase.shouldGrant)
```